### DISCUSSION TITLE
Ability to use other training/models for music generation?

### DISCUSSION THREAD

**Discussion by ghost** [intel/openvino-plugins-ai-audacity#384] [Q&A]
Hello,

Not so much an issue, but a question - and please feel free to let me know if this isn't the appropriate forum, but is there a way to implement other training models for use with the Music Generation portion of OpenVino?

Either way, this is great tool - thank you!

John

**Comment by MarcoRavich** [comment]
...what about posting it into [Discussions](https://github.com/intel/openvino-plugins-ai-audacity/discussions), then ?
